waspc/e2e-test/GoldenTest.hs

```diff
@@ -83,3 +99,24 @@ runGoldenTest goldenTest = do
     writeFileManifest baseAbsFp filePaths manifestAbsFp = do
       let sortedRelativeFilePaths = unlines . sort . map (makeRelative baseAbsFp) $ filePaths
       writeFile manifestAbsFp sortedRelativeFilePaths
+
+    -- This function normalizes all package.json files for comparison as `npm install` can overwrite
+    -- them when creating/updating package-lock.json. Different versions of Node may produce different
```

Comment:
Should that be different versions of npm? (which come with different versions of node, and it's not a good idea to start varying them independently I've found, but still)

Reply:
Good point, I can change Node to say NPM for clarity. 👍🏻